pages/common/zmv.md

```diff
@@ -0,0 +1,28 @@
+# zmv
+
+> Move or rename files matching the regex pattern to corresponding files having names of the form given by destination.
+> More information: <http://zsh.sourceforge.net/Doc/Release/User-Contributions.html>.
+
+- Below renames `foo.log` to `foo.txt`, and so on:
```

Comment:
Is there any way  that we can describe precisely what this is doing, rather than giving an example of how the example functions here?

Reply:
I think examples are better than full-on descriptions but if there are any specific changes you'd like, I'm happy to incorporate that. I'm also wary of not adding too much text as `tldr` :P

Reply:
Hrm. What about this?
```suggestion
- Change extensions in filenames using a pattern:
```

Reply:
It may give the users an impression that it can only be used for file ext change. 

How about the below example?

`zmv '{{(*) - (*).txt}}' '{{$2 - $1.txt}}'`

This renames `test - item.txt` to `item - test.txt`

This is how I use it mostly and that's where I see its strength.

Reply:
A better example is here https://github.com/tldr-pages/tldr/pull/4705#discussion_r507267446

Reply:
The first example should be the simplest (see [guideline #4](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines)). We can add a second example with more advanced regex, to showcase the power of the tool, but the first one should be short and easy to understand at a glance. I think we can keep the current example and just change the description, maybe to something like this:

```
- Move files using a regex-like pattern:

`zmv '{{(*).log}}' '{{$1.txt}}'`
```

WDYT?